README.md

```diff
@@ -77,102 +73,103 @@ Or in order to install to another root directory:
 
     $ sudo DESTDIR=/tmp/root/dir ninja -C build install
 
-### Running ###
 
-To route mavlink packets from master `ttyS1` to 2 other UDP endpoints, do as
-following:
+## Running
 
-    $ mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550 /dev/ttyS1:1500000
+There are two ways to configure mavlink-router which can be used in combination: Configuration file(s) and command line parameters.
+The configuration file has a more fine-grained control (and is used by the systemd unit) while the CLI options enable a quick setup of endpoints.
 
-The `1500000` after the colon in `/dev/ttyS1:1500000` sets the
-UART baudrate. See more options with `mavlink-routerd --help`.
+### Conf File
 
-It's also possible to route mavlinks packets from any interface using:
+By default, mavlink-routerd looks for a file `/etc/mavlink-router/main.conf`.
+The file location can be overriden via a `MAVLINK_ROUTER_CONF_FILE` environment variable, or via the `-c` CLI switch when running mavlink-routerd.
+An example conf file can be found in [examples/config.sample](examples/config.sample)
 
-    $ mavlink-routerd -e 192.168.7.1:14550 -e 127.0.0.1:14550  0.0.0.0:24550
+### Conf Directory
 
-mavlink-router also listens, by default, on port 5760 for TCP connections. Any
-connection there will also receive routed packets.
+Besides the default conf file, it's also possible to use a directory in where to put some extra configuration files.
+Files in this directory will be read in alphabetical order, and can add or override configurations found in previous files.
 
-IPv6 addresses must be enclosed in square brackets like this: `[::1]`. The port number
-can be specified in the same way, as with IPv4 then: `[::1]:14550`.
+By default, `/etc/mavlink-router/config.d` is the directory, but it can be overriden via a `MAVLINK_ROUTER_CONF_DIR` environment variable, or via the `-d` switch when running mavlink-routerd.
 
-<a name="Conffiles"></a>
-### Conf file ###
+### CLI Parameters
 
-It's also possible to use a .conf file to set options for mavlink-routerd.
-By default, mavlink-routerd looks for a file
-`/etc/mavlink-router/main.conf`. The file location can be overriden via a
-`MAVLINK_ROUTER_CONF_FILE` environment variable, or via the `-c` switch when running
-mavlink-routerd.
-An example conf file can be found in [examples/config.sample](examples/config.sample)
+Please see the output of `mavlink-routerd --help` for the full list of command line options.
+The most important facts are:
 
-#### Conf dirs ####
+- The TCP server is enabled by default
+- TCP and UDP endpoints can be added multiple times
+- UDP endpoints added with the `-e` option are started in `normal` mode (sending data to the specified address and port)
+- The last parameter (without a key) can either be one UART device or a UDP connection.
+This UDP endpoint will be started in `server` mode (waiting for an incoming connection)!
```

Comment:
we probably can say this is just for compatibility with previous behavior. Because without a concept of "master", the additional arg is not really needed

Reply:
But it's still a different behavior for the UDP endpoints (server vs normal mode), right?

Reply:
indeed... maybe that's why it was left there.